<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreInfo</class>
 <widget class="QWidget" name="StoreInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1343</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(33, 33, 33);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="styleSheet">
        <string notr="true">color: white;
background-color: rgb(148, 214, 218);</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,1">
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_3">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                 <number>15</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QComboBox" name="queryTypeBox">
                    <property name="minimumSize">
                     <size>
                      <width>160</width>
                      <height>37</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>160</width>
                      <height>37</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>16</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color:#ffffff;
background-color: rgb(0, 166, 172);
QComboBox::drop-down {
    subcontrol-position: top right;
    width: 30px;
    border-left-width: 1px;
    border-left-style: solid; 
	border-left-color: gray;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}</string>
                    </property>
                    <item>
                     <property name="text">
                      <string>货架编号</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>工单编码</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>胶箱编码</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="queryMaterial">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>20</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgb(45, 45, 45);
background-color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string>A01</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_count">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_4">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="spacing">
                 <number>15</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="font">
                     <font>
                      <pointsize>16</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color:#ffffff;
border: 0px groove gray;</string>
                    </property>
                    <property name="text">
                     <string>路径:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="queryMaterial_2">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>16</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(255, 255, 255);</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="lead_Button">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>160</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: white;
color:#ffffff;
border-style: outset;
border: 2px groove gray; 
border-radius:10px;
background-color: rgb(81,81,81);</string>
                  </property>
                  <property name="text">
                   <string>导入改仓EXcel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <widget class="QPushButton" name="query_Button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>45</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>160</width>
                <height>45</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;
color:#ffffff;
border-radius:15px;
border-style: outset;
border: 2px groove gray; 
background-color: rgb(0, 166, 172);</string>
              </property>
              <property name="text">
               <string>查询</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(66, 66, 66);
color: rgb(33, 33, 33);</string>
           </property>
           <column>
            <property name="text">
             <string>序号</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>物料件号</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>仓号</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Bin</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>批次</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>数量</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>基础单位</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,1">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QStackedWidget" name="stackedWidget_3">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(189, 189, 186);</string>
             </property>
             <widget class="QWidget" name="page_5"/>
             <widget class="QWidget" name="page_6"/>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 102, 102);
background-color: rgb(141, 131, 130);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
